test/simple.test.js

```diff
@@ -5,33 +5,46 @@ const squid = require('flying-squid')
 
 const settings = require('../config/default-settings')
 
-describe('server', () => {
-  let serv
-
-  beforeAll(done => {
-    const options = settings
-    options['online-mode'] = false
-    options['port'] = 25566
-    options['view-distance'] = 2
-    options['worldFolder'] = undefined
-    options['logging'] = false
-    serv = squid.createMCServer(options)
-
-    serv.on('listening', () => {
-      done()
+const { firstVersion, lastVersion } = require('./common/parallel')
+
+squid.supportedVersions.forEach((supportedVersion, i) => {
+  if (!(i >= firstVersion && i <= lastVersion)) {
+    return
+  }
+  const PORT = Math.round(30000 + Math.random() * 20000)
```

Comment:
This doesn't smell good

Reply:
why ?
This is what I do in mineflayer and node-minecraft-protocol. It allows for running multiple tests at the same time without any port collision

Reply:
You should use the `CIRCLE_NODE_INDEX` environment variable instead, then add that to `3000`, for example.

Reply:
Ah yeah I could do that. It's unlikely to be a problem but why not. I'll change that

Reply:
No, that won't work locally.

Reply:
`process.env.CIRCLE_NODE_INDEX || 0`

Reply:
Or even `|| (Math.random() * 20000)`

Reply:
fixed, going full deterministic

Reply:
used port 0. Much more reliable. Thanks for insisting on that issue !